mobile-widgets/qml/Settings.qml

```diff
@@ -622,7 +622,7 @@ TemplatePage {
 					checked: PrefTechnicalDetails.dcceiling
 					onClicked: {
 						PrefTechnicalDetails.dcceiling = checked
-						rootItem.settingChanged()
+						rootItem.settingsChanged()
```

Comment:
UGH. I hate that there isn't better syntax chcking... I have had so many errors like this over the years in the QML code :-(